microsoft-365/security/office-365-security/virus-detection-in-spo.md

```diff
@@ -21,9 +21,9 @@ description: Learn about how SharePoint Online detects viruses in files that use
 ms.custom: seo-marvel-apr2020
 ---
 
-# Virus detection in SharePoint Online
+# Virus detection in SharePoint Online, OneDrive and Microsoft Teams
 
-Microsoft 365 can help protect your environment from malware by detecting viruses in files that users upload to SharePoint Online. Files may be scanned for viruses after they are uploaded. If a file is found to be infected, a property is set so that users can't download or sync the file.
+Microsoft 365 can help protect your environment from malware by detecting viruses in files that users upload to SharePoint Online, OneDrive and Microsoft Teams. Files may be scanned for viruses after they are uploaded. If a file is found to be infected, a property is set so that users can't download or sync the file.
```

Comment:
```suggestion
Microsoft 365 can help protect your environment from malware by detecting viruses in files that users upload to SharePoint Online, OneDrive, and Microsoft Teams. Files may be scanned for viruses after they are uploaded. If a file is found to be infected, a property is set so that users can't download or sync the file.
```